client/css/style.css

```diff
@@ -2243,6 +2256,12 @@ part/quit messages where we don't load previews (adds a blank line otherwise) */
 		padding: 5px 10px;
 	}
 
+	#chat .channel .message.highlight::before {
+		margin: 0;
+		height: 12px;
+		min-width: 12px;
```

Comment:
I wanted to leave this at 16px, but I get some weird alignment issues beyond 12px (even though the text is supposed to be 18px high, I don't entirely understand).